app.js

```diff
@@ -410,24 +442,42 @@ const tslint = {
     printDone('Installing HammerJS...');
   }
 
+  if (answers.git) {
+    printMsg('Linkining to git...');
+    try {
+      version = cp.execSync(`git --version`);
+    } catch (error) {
+      printDone('git is not installed...');
+    }
+    if (version && !!version.toString()) {
+      cp.execSync(`git init`);
```

Comment:
ng new already creates a git repo by default, but we could add a question if you want to generate a git repo when creating the angular app
https://angular.io/cli/new
--commit=true|false